lib/git_reflow.rb

```diff
@@ -69,32 +71,34 @@ def deliver(options = {})
     update_destination(current_branch)
 
     begin
-      existing_pull_request = git_server.find_pull_request( :from => current_branch, :to => options['base'] )
+      existing_pull_request = git_server.find_open_pull_request( :from => current_branch, :to => options['base'] )
 
       if existing_pull_request.nil?
-        puts "Error: No pull request exists for #{remote_user}:#{current_branch}\nPlease submit your branch for review first with \`git reflow review\`"
+        say "No pull request exists for #{remote_user}:#{current_branch}\nPlease submit your branch for review first with \`git reflow review\`", :deliver_haulted
```

Comment:
`deliver_halted` would be the right way to spell it, not sure if that's correct for this